tests/storage-vm

```diff
@@ -343,13 +343,13 @@ for poolDriver in $poolDriverList; do
         lxc exec v1 -- rm /srv/rw/lxd-test-rw
         lxc exec v1 -- rm /srv/rw/lxd-test
 
-        # Check block disks are available.
-        lxc exec v1 -- stat -c "%F" /dev/sdb | grep -xF "block special file"
-        lxc exec v1 -- stat -c "%F" /dev/sdc | grep -xF "block special file"
+        # Check /dev/disk/by-id/ links
+        lxc exec v1 -- test -b "$(lxc exec v1 -- readlink -f /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_lxd_block1ro)"
```

Comment:
I don't think we need to do the symlink deference manually, do we?

```suggestion
        lxc exec v1 -- test -b /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_lxd_block1ro
```

Reply:
fixed, thanks for the insight!